StripeCore/StripeCore/Source/API Bindings/StripeAPI.swift

```diff
@@ -68,9 +68,33 @@ import PassKit
             }
         }
     }
+
+    /// Users that have the Payment Method Cartes Bancaires set to Active
+    /// can enable Cartes Bancaires for Apple Pay by setting this to `YES`.
+    ///
+    /// The default value is NO.
+    @objc public class var cartesBancairesPaymentNetworkSupported: Bool {
+        get {
+            return self.additionalEnabledApplePayNetworks.contains(.cartesBancaires)
+        }
+        set(CartesBancairesPaymentNetworkSupported) {
+            if CartesBancairesPaymentNetworkSupported
+                && !self.additionalEnabledApplePayNetworks.contains(.cartesBancaires)
+            {
+                self.additionalEnabledApplePayNetworks = [PKPaymentNetwork.cartesBancaires] +
+                    self.additionalEnabledApplePayNetworks
```

Comment:
Yes, we try to go through CB before other gateways, because it's usually more cost effective.
We currently prepend Cartes Bancaires to the list of supported networks on Apple Pay web. 
The outcome here still isn't ideal, because `additionalEnabledApplePayNetworks` is appended to the "default" networks list [here](https://github.com/stripe/stripe-ios/blob/da89a2cd/StripeCore/StripeCore/Source/API%20Bindings/StripeAPI.swift#L109).
Ideally we would prepend CB to the whole list of supportedPKPaymentNetworks, but this is out of the scope of this PR may be a future optimization.